.github/workflows/cmake_conan.yml

```diff
@@ -22,4 +22,6 @@ jobs:
         with:
           cmake-version: '3.25.x'
       - name: Run Tests
-        run: pytest -rA -v
+        run: |
+          cmake -S tests -B build_unittests
```

Comment:
I think it would be better to keep all tests under the ``pytest`` framework.
It could be a test that is just a line that ``run("cmake ....")``, but also better to run them locally, etc.

Reply:
Sure, I can do that, any preference on adding it to `test_smoke.py` versus creating a new `test_unittest.py`?

Reply:
A new ``test_conan_version.py`` sounds good.